ownCloudAppShared/Intent/Extensions/OCLicenseManager+Setup.swift

```diff
@@ -46,19 +48,22 @@ public extension OCLicenseManager {
 		let documentScannerFeature = OCLicenseFeature(identifier: .documentScanner, name: "Document Scanner".localized, description: "Scan documents and photos with your camera.".localized)
 		let shortcutsFeature = OCLicenseFeature(identifier: .shortcuts, name: "Shortcuts Actions".localized, description: "Use ownCloud actions in Shortcuts.".localized)
 		let documentMarkupFeature = OCLicenseFeature(identifier: .documentMarkup, name: "Markup Documents".localized, description: "Markup photos and PDF files.".localized)
+		let photoProFeature = OCLicenseFeature(identifier: .photoProFeatures, name: "Photo Pro Features".localized, description: "Image metadata, extended upload options".localized)
 
 		// - Features
 		register(documentScannerFeature)
 		register(shortcutsFeature)
 		register(documentMarkupFeature)
+		register(photoProFeature)
 
 		// - Single feature products
 		register(OCLicenseProduct(identifier: .singleDocumentScanner, name: documentScannerFeature.localizedName!, description: documentScannerFeature.localizedDescription, contents: [.documentScanner]))
 		register(OCLicenseProduct(identifier: .singleShortcuts, name: shortcutsFeature.localizedName!, description: shortcutsFeature.localizedDescription, contents: [.shortcuts]))
 		register(OCLicenseProduct(identifier: .singleDocumentMarkup, name: documentMarkupFeature.localizedName!, description: documentMarkupFeature.localizedDescription, contents: [.documentMarkup]))
+		register(OCLicenseProduct(identifier: .singlePhotoProFeatures, name: photoProFeature.localizedName!, description: photoProFeature.localizedDescription, contents: [.photoProFeatures]))
 
 		// - Subscription
-		register(OCLicenseProduct(identifier: .bundlePro, name: "Pro Features".localized, description: "Unlock all Pro Features.".localized, contents: [.documentScanner, .shortcuts, .documentMarkup]))
+		register(OCLicenseProduct(identifier: .bundlePro, name: "Pro Features".localized, description: "Unlock all Pro Features.".localized, contents: [.documentScanner, .shortcuts, .documentMarkup, .photoProFeatures]))
```

Comment:
Just quickly glossed over the code setting up licensing for the photo pro features and noticed that one piece of the puzzle is still missing: if Photo Pro Features should be available as a one-time purchase, too, its app store non-consumable IAP would need to be provided to `OCLicenseAppStoreProvider` a few lines down from here, in `OCLicenseManager+Setup.swift:71`++.

Reply:
@felix-schwarz good catch.. indeed it was missing.. fixed